content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
+* **Live reload** is a convenience function that many themes will include as part of their build tooling. It simply means that when you edit source files after they are recompiled, your browser will magically refresh and display the changes without a manual refresh.
 
- * `email` is your default email address. If your theme has an option to send an email, then you can find it on default parameters.
+## What are the source, compiled, and static files?
 
- * `address` is your default address. If your theme has an option to show your address, then you can find it on default parameters.
+These terms are can mean different things in different contexts, but for the purposes of a Hugo Theme:
 
- * `search` doesn't comes with every theme. But if your theme has a search bar, and if you don't want it, then you can disable it by making it `search = false`.
+* **Source files** are files that are meant to be processed by a theme’s build tools.
+* **Compiled files** are generated as a result of running a compiling process (also called a "build process") 
+on the source files.
+* **Static files** are ones that aren’t processed or generated.
 
- * `navigation_button` is for the main navigation button. Many themes have a button on navigation that target contact or other pages. If your theme has this button, then you can handle this button `label` and `link` from here, or if you don't want to show it, then you can disable it by `enable = false`.
+**Here are a few real-world examples of each type of file:**
+
+* **SCSS files always source files** because they must be compiled by your theme’s build tool to generate a CSS file that a browser can understand. SCSS files are located in `assets` folder.
+* **HTML files with any non-standard syntax are source files.** Some themes make use of `partials`, `shortcodes`, `internal templates` to create elements. Those must be compiled by `Hugo` to generate standard HTML. Those HTML files are located in `layouts` folder.
+* **exampleSite folder has source files.** This folder includes all `content`, `data`, and `config` files which must be compiled to run on the browser.
+* **Fonts, images, and plugins are static files.** These don’t require any processing to be used and aren’t generated from any other source. Fonts and Plugins are located in `static` folder. And Images are located in `exampleSite/static` folder because images need to change when you build a real site.
+* **The output of those above examples are compiled files.** The CSS files are generated by processing the source SCSS. The HTML files are generated by processing the pseudo-HTML source. And all pages are generated from exampleSite contents. All are generated from source files and will be overwritten if the source is compiled again. Output files can be found on `exampleSite/public` folder, when you run `hugo --themesDir ../..` this command. If you want the minified output, then run `hugo --minify --themesDir ../..`.
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)